Plugins/Json/include/Acts/Plugins/Json/JsonGeometryConverter.hpp

```diff
@@ -1,19 +1,17 @@
 // This file is part of the Acts project.
 //
```

Comment:
Also here, we are mixing

`ThingJsonConverter.hpp` and `JsonThingConverter.hpp` - I am good either way, but after the `SurfaceJsonConverter.hpp` is in, you should either synchronize either.

Reply:
I agree I should change this name. Let me thing of something better

Reply:
I changed it to MapJsonConverter